frontend/src/metabase/common/components/EntityPicker/components/SearchTab/SearchResults.tsx

```diff
@@ -0,0 +1,71 @@
+import { t } from "ttag";
+
+import EmptyState from "metabase/components/EmptyState";
+import { VirtualizedList } from "metabase/components/VirtualizedList";
+import { NoObjectError } from "metabase/components/errors/NoObjectError";
+import { trackSearchClick } from "metabase/search/analytics";
+import { Box, Flex, Stack } from "metabase/ui";
+import type { SearchResult, SearchResultId } from "metabase-types/api";
+
+import type { TypeWithModel } from "../../types";
+import { ChunkyList, ResultItem } from "../ResultItem";
+
+interface Props<
+  Id extends SearchResultId,
+  Model extends string,
+  Item extends TypeWithModel<Id, Model>,
+> {
+  searchResults: SearchResult[];
+  selectedItem: Item | null;
+  onItemSelect: (item: Item) => void;
+}
+
+export const SearchResults = <
```

Comment:
This component has been moved from `EntityPickerSearch.tsx`.
What's been changed:
- Removed `EntityPicker` prefix 
- `DelayedLoadingSpinner` has been lifted to `SearchTab`
- `searchResults` prop can't be `null` now
- `Props` are no longer defined inline

The rest is the same.